frontend/components/footer/flex/FooterFlexCol.vue

```diff
@@ -48,10 +48,12 @@
               :href="connect.url"
               target="_blank"
             >
-              <Icon :name="connect.iconName" class="text-2xl sm:text-base" />
-              <p class="sr-only sm:not-sr-only">
-                {{ $t(connect.name) }}
-              </p>
+              <MetaTagSocialMedia
+                class="text-2xl sm:text-base"
+                :iconName="connect.iconName"
+                :text="$t(connect.name)"
+                textStyleClass="sr-only sm:not-sr-only"
```

Comment:
I'll rename the prop to `textClassUtilities` as it's a bit more descriptive of Tailwind classes. As it stands it looks more like it's for styling the text like passing color or size props, which isn't what we're using it for as of now 😊 Really great use of props though, @SukhOberoi!